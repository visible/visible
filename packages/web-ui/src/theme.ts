export interface Theme {
  foreground: {
    normal: string;
    wash: string;
  };
  background: {
    normal: string;
    wash: string;
  };
  highlight: {
    normal: string;
  };
  border: {
    normal: string;
  };
}
