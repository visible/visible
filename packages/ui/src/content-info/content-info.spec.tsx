import { render } from '@testing-library/react';
import React from 'react';
import { ThemeProvider } from 'styled-components';

import { theme } from '../theme';
import { ContentInfo } from '.';

describe('ContentInfo', () => {
  it('matches snapshot', () => {
    const { container } = render(
      <ThemeProvider theme={theme}>
        <ContentInfo />
      </ThemeProvider>,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
