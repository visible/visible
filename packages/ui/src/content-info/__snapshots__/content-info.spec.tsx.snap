// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentInfo matches snapshot 1`] = `
.c0 {
  -webkit-transition: 0.15s ease-out;
  transition: 0.15s ease-out;
  opacity: 0.5;
  font-size: 12px;
}

.c0 a {
  color: #666666;
}

.c0 .sc-AykKC {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  width: 1080px;
  margin: auto;
  padding: 0;
  list-style: none;
}

.c0:hover,
.c0:focus {
  -webkit-transition: 0.15s ease-in;
  transition: 0.15s ease-in;
  opacity: 1;
}

<footer
  class="c0"
/>
`;
